### ISSUE TITLE
New spago args (--json-errors)

### ISSUE THREAD

**Issue by wclr** [nwolverson/purescript-language-server#206]
This is just in case. New spago assumes passing `--json-errors` without `--purs-args`: `spago build --json-errors`, currently by default LS uses `spago build --purs-args --json-errors` build command wich gives an error. 

So one can use `buildCommand` option to override the default build command.

**Comment by nwolverson** [comment]
Yes, this for sure should become the default eventually, I guess new spago is still officially alpha, not sure what the usage % now splits at. That's assuming it isn't supported in old-spago without `--purs-args`
